app.js

```diff
@@ -31,15 +30,25 @@ const logger = new winston.Logger({
 	exitOnError: false
 });
 
-// all environments
-app.set('port', process.env.PORT || 3000);
 app.set('views', __dirname + '/views');
 app.set('view engine', 'ejs');
 app.engine('html', require('ejs').renderFile);
-app.use(bodyParser.urlencoded({ extended: true }));
-app.use(bodyParser.json());
-app.use(methodOverride());
 app.use(express.static(path.join(__dirname, 'public')));
+
+// if this app is the primary app (either has no parent or can be set in env)
+if (env.standalone) {
```

Comment:
Comment in line 38 "either has no parent" doesn't match the if statement after.  I think it's OK to do what the comment says as long as the env variable takes precedence.